# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsPIDOMWindow.h
# Commit: 812ed5275f0d
# Full Hash: 812ed5275f0db9f92944955f891df2ab68c9f8e7
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-11-19 04:11:43
# Regressor Bug: 1734099
# File Overlap Count: 5
# Description:
#   Bug 1734099 - Never unload tabs with active RTCPeerConnection instances.  r=peterv,jesup
#   
#   This patch makes sure the Tab Unloading feature does not unload tabs that have
#   active peer connections not to disrupt browsing experience based on WebRTC peer
#   connections.
# ==============================================================================

diff -r a733d049dbda -r 812ed5275f0d dom/base/nsPIDOMWindow.h
--- a/dom/base/nsPIDOMWindow.h	Fri Nov 19 01:03:56 2021 +0200
+++ b/dom/base/nsPIDOMWindow.h	Thu Nov 18 22:59:12 2021 +0000
@@ -645,10 +645,6 @@
 
   uint32_t mActivePeerConnections = 0;
 
-  // This is the count for active peer connections for all the windows in the
-  // subtree rooted at this window (only set on the top window).
-  uint32_t mTotalActivePeerConnections = 0;
-
   bool mIsDocumentLoaded;
   bool mIsHandlingResizeEvent;
   bool mMayHavePaintEventListener;